README.md

```diff
@@ -0,0 +1,111 @@
+<a href="https://ingot.gg/">
+    <p align="center">
+        <img width="325" height="325" src="https://raw.githubusercontent.com/IngotGG/branding/master/branding.svg" alt="iron"/>
+    </p>
+</a>
+
+<p align="center">
+    <strong>Iron, a simple JDBC Wrapper.</strong>
+</p>
+
+--- 
+
+# Iron [![](https://jitpack.io/v/gg.ingot/iron.svg)](https://jitpack.io/#gg.ingot/iron)
+
+Iron is a simple yet powerful JDBC Wrapper used by [ingot.gg](https://ingot.gg) backend services to interact
+with our SQL Databases.
+
+## Importing
```

Comment:
Reference the jitpack link in this section with a hyperlink pointing to the page, alongside possibly add the jitpack badge so people know what version to use (unless we want to do releases)